### ISSUE TITLE
Done

### ISSUE THREAD

**Issue by RobinMalfait** [gcedo/react-solitaire#5]
Now what? :D

![](http://d.pr/i/1aOXl+)

**Comment by gcedo** [comment]
Now you behold the beauty of react, while savoring the glory of victory :smile: 

Yep, I need to add a winning modal at least.

**Comment by RobinMalfait** [comment]
Good job though, I finished it at the first time, so not sure if it is intended :p

**Comment by gcedo** [comment]
Not at all, I simply use [lodash `shuffle` function](https://lodash.com/docs#shuffle) to shuffle the cards :blush:

**Comment by RobinMalfait** [comment]
Okay, that said, I nailed it :smile:
